app/reducers/events.js

```diff
@@ -327,7 +330,7 @@ export const selectAllRegistrationsForEvent = createSelector(
           updatedBy,
         });
       })
-      .filter((reg) => reg.event == eventId)
+      .filter((reg) => reg.event === eventId)
```

Comment:
:shipit:

Reply:
Rip `int` and `string` comparison. Fixed by wrapping `eventId` in `parseInt()`